Add OpenPBR scene variants

theblackunknown opened the pull request:
## Summary

This PR fixes #91 by adding a `material_model` variant set on various prims to select an OpenPBR surface shader instead of the existing surface shader.
We keep backward compatibility by setting `material_model` variants set to `standard_surface` to keep the legacy behavior and look.

For easy consumption a new "top layer" stage file `standard_shader_ball_scene_openpbr.usda` has been added.

Note that this PR keeps the current ways of specifying looks: when a MaterialX document was used, I have added another one for OpenPBR tagged with MaterialX v1.39, where `UsdShade` graphs were used I have simply added new nodes.
Ideally we should strive to create MaterialX document for `mtlx` nodegraphs so that MaterialX data versionning and upgrading cna be handled via `usdMtlx`.

Quick test of using `standard_shader_ball_scene_openpbr.usda` in Maya and trying some variants from `example_material` variant set
![image](https://github.com/user-attachments/assets/7e4c9d0a-d974-4404-8790-73bd398f9c22)

crydalch commented:
Thanks for starting this up, great start! Some thoughts:

- We should be sure to name the variant "OpenPBR Surface" or "OpenPBRSurface" (i.e. we should include "surface" not just "OpenPBR")
- I'm not sure how I feel about having a separate layer that just sets the shading variant? It's easy enough to switch variants in usdview, and I don't feel like there's a lot of benefit to using variants, if we start adding separate layers that only set variants.
- I think having OpenPBR Surface be the default makes sense; though it might be a bit premature, as mtlxstandardsurface is more widely supported to this point... Would be good to hear from others in the community on that point

Just some thoughts, but thanks again for starting this!

theblackunknown commented:
Thanks for your review @crydalch !

> * We should be sure to name the variant "OpenPBR Surface" or "OpenPBRSurface" (i.e. we should include "surface" not just "OpenPBR")

Good point, I have updated this (a8e705cd87ca4093bdb763c51a634adf1fae5b52)

> * I'm not sure how I feel about having a separate layer that just sets the shading variant? It's easy enough to switch variants in usdview, and I don't feel like there's a lot of benefit to using variants, if we start adding separate layers that only set variants.

This was originally brough up as a convenience for all those applications which does not have good variant support.
I do not have a strong opinion on this, so I'll remove it for now. (c4de7e0e168046f44b5d8770b855804a73ac8a56) 

> * I think having OpenPBR Surface be the default variant selection could makes sense; though it might be a bit premature, as mtlxstandardsurface is more widely supported to this point... Would be good to hear from others in the community on that point.

I am personally fine to make the switch now, especially as we can have backward compatibility by a simple variant switch. (577a8c194aed12740028ba05cd8c8ff1e95f3c7b)

What do you think ?

> * We should also probably increase the version as well in the readme, to 1.1

Very good point, I have added it and other information about the update in the README (34b042b0a2015a6cd85e43e6e40ff383bcca3123)
